### ISSUE TITLE
Destination Calendar list don't sync with my Calendar list

### ISSUE THREAD

**Issue by JOduMonT** [SergeyMosin/Appointments#335]
## version
- Nextcloud v22.xx (latest)
- Calendar v 3.3.0
- Appointments v1.12.4

Obviously I played, probably too much, with the Appointment App, before looking the tutorial, and I'm looking for a way to reset it.
I tried to disable, remove and reinstall it, but the bug remain; I probably need to do an `occ` command and I could but which one ?

So just to give an idea
### in Appointment
![image](https://user-images.githubusercontent.com/5204724/169191497-710cf23a-1f1e-4fdd-a3de-2db8aacc4f05.png)
I see several time _Personal_ (which I don't know why) but not the Calendar I create for Appointment

### in Calendar
![image](https://user-images.githubusercontent.com/5204724/169191681-badae59f-6255-4ab1-9c32-d4808349e669.png)

It seems to be out of sync.

**Comment by stefan-franz** [comment]
Hello, i have also 3 Personal entrys - maybe a bug in he Appointment App?

Try out every entry - in my case the lowerst was the right.

**Comment by SergeyMosin** [comment]
Double listing bug is probably related to #64 and should be fix in the next release.

@JOduMonT Assuming you are looking for the `Deck: Business` calendar can you please check that it is not **read-only** or **subscription/remote** calendar.

`Settings > Advanced Settings > Debugging > Get raw calendar data` which would look something like this:
```
array (
  'name' => 'appt_weekly',
  'id' => '3',
  'isReadOnly' => '0',
  'isSubscription' => '0',
)

array (
...
```  
If `isReadOnly` or `isSubscription` is set to `1` then this calendar can not be used. The `read-only` status is usually set when a calendar is shared and `subscription` calendars can not be used because they are read-only by default.

**Comment by SergeyMosin** [comment]
#64 is fixed in v1.12.5
